TypeCobol.Analysis/Cfg/ControlFlowGraphBuilder.MultiBranchContext.cs

```diff
@@ -35,28 +31,49 @@ internal class MultiBranchContext
             /// The Index in the SuccessorEdge of the RootBlock if any, -1 otherwise.
             /// </summary>
             internal int RootBlockSuccessorIndex;
-
+            /// <summary>
+            /// Terminals block associated to Multi branch Context if any.
+            /// Terminals blocks are block that have as successor NextFlowBlock or
+            /// a branching to the beginning of a loop instruction block.
+            /// </summary>
+            public IList<BasicBlock<Node, D>> Terminals { get; internal set; }
```

Comment:
Do we need to compute that in the parser ? As it is a new property, I think it will be used by our private analyzer but it is not required by the CFG/DFA itself. So maybe we could compute terminals externally ? So as to reduces changes in the parser as much as possible.

Reply:
Yes Terminals value are calcualted at multi branch context ending step see End(...) method.

Reply:
In fact this looks like a duplicate with the existing `GetTerminalSuccessorEdges` method. Could you merge the two ?

Reply:
Not really because at the End()  method time Terminals on those calculated at construction time, the whole CFG is not built, but at GetTerminalSuccessorEdges time the whole Graph may have  been built and terminals can refer to terminals block that are not those of the multi branch context.